login_page/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>login_page</title>
+    <link rel="stylesheet" href="/reset.css"/>
+    <link rel="stylesheet" href="/login_page/style.css" />
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div class="container">
+        <a class="panda_logo" a href="/index.html">
+            <picture>
+                <source media="(max-width: 768px)" srcset="/login_page/login_image/mobile/panda_logo_mobile.png">
+                <img src="/login_page/login_image/logo.png" alt="판다마켓 로고">
+            </picture>
+        </a>
+        <div class="input_sector">
+            <div class="email_sector">
+                <span>이메일</span>
+                <div class="inputbox">
+                    <input type="email" placeholder="codeit@email.com" >
+                </div>
+            </div>
+            <div class="pw_sector">
+                <span>비밀번호</span>
+                <div class="inputbox">
+                    <input type="password" placeholder="">
+                    <div class="eye">
+                        <i class="fas fa-eye"></i>
+                    </div>
```

Comment:
❗️ 수정요청  
해당 요소는 클릭 시 비밀번호가 보이고 안 보이게 만드는 요소가 될 것이므로 button요소로 작성하시는 것을 추천드려요.

```suggestion
                    <button type="button" class="eye">
                        <i class="fas fa-eye"></i>
                    </button>
```